specify/specify7
# Add series data entry feature

Fixes #545 

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- Create a new CO 
- Save 
- Go in the form meta, check the box "Show Bulk Carry Forward count" 
- [ ] Verify the box "Show Bulk Carry Forward count" is displayed only when carry forward is checked 
- [ ] Verify that the Bulk amount input is not displayed on the form if carry forward or "Show Bulk Carry Forward count" is not checked in the form meta
- Go back to the CO form 
- Enter the amount you want to bulk carry 
- Click on carry forward 
- [ ] Verify that a new RS is created 
- [ ] Verify that the new RS has, as a title, the cat number range 
- [ ] Verify once the new RS is created, it opens it
